Cubical/Codata/M/AsLimit/Coalg/Properties.agda

```diff
@@ -0,0 +1,207 @@
+{-# OPTIONS --cubical --guardedness #-} --safe
```

Comment:
safe?

Reply:
I have not completed all the postulates, but will make it safe, once i do.